tools/onnx/sarif/gen_sarif.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Run this script inside its folder to generate the SARIF python object model files
+# from the SARIF schema.
+# e.g. sh gen_sarif.sh
+#
+# This script requires the jschema_to_python package to be installed.
+# To install it, run:
+#   pip install jschema_to_python
+ROOT=$(pwd)/../../..
+SARIF_DIR=torch/onnx/_internal/diagnostics/infra/sarif_om
+
+# SARIF version
+SARIF_VERSION=2.1.0
+SARIF_SCHEMA_LINK=https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/schemas/sarif-schema-2.1.0.json
+
+# Download SARIF schema
+tmp_dir=$(mktemp -d)
+sarif_schema_file_path=$tmp_dir/sarif-schema-$SARIF_VERSION.json
+wget -O $sarif_schema_file_path $SARIF_SCHEMA_LINK
+
+# TODO: A private branch of jschema_to_python was used to enable
```

Comment:
why was the private needed? If anyone else needs to run this, which branch they should use (full path is better)? Is the branch `0.0.1.dev29` from upstream?

Reply:
Oops this is leaked down to this commit. Only the follow ups that generate code with dataclasses and type annotations need this. A private branch is needed because official one only supports attr, which PyTorch does not. And afaik there is no other tools on the market to generate from jschema to python.